cli/index.js

```diff
@@ -39,13 +43,355 @@ const signUp = (args) => {
 
             const FBIdToken = `Bearer ${res.data.token}`;
             // TODO: create a file in the user's machine to store the auth token
-            console.log(chalk.white.bold("Account created successfully ✔️"));
+            console.log(chalk.green.bold("Account created successfully ✔️"));
         })
         .catch((error) => {
             console.log(chalk.red.bold(`Could not create account -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
         });
 }
 
+// *definition of create guest command
+const guest= (args) => {
+
+    const { userId, picture, username } = args;
+
+    const guestuserdata= {
+        userId,
+        picture,
+        username,
+    };

```

Comment:
It's great to specify what action you are doing to the guest, a function name of 'guest' doesn't really tell much. What is it, guestSignUp?

Reply:
Done